core/src/main/java/org/mapfish/print/map/style/json/MapfishJsonStyleVersion2.java

```diff
@@ -27,7 +27,7 @@
 public final class MapfishJsonStyleVersion2 {
   static final String JSON_SYMB = "symbolizers";
   private static final String JSON_TYPE = "type";
-  private static final Pattern VALUE_EXPR_PATTERN = Pattern.compile("\\$\\{([\\w\\d_-]+)\\}");
+  private static final Pattern VALUE_EXPR_PATTERN = Pattern.compile("\\$\\{([\\w_-]+)\\}");
```

Comment:
Why remove '\\d'?

Reply:
\d was redundant:
\w\d is the same as \w